system/jlib/jexcept.cpp

```diff
@@ -1452,6 +1452,44 @@ void printStackReport(__int64 startIP)
 
 //---------------------------------------------------------------------------------------------------------------------
 
+bool getAllStacks(StringBuffer &output)
+{
+#ifdef __linux__
+    StringBuffer exePath;
+    char *_exePath = exePath.reserveTruncate(NAME_MAX + PATH_MAX +1);
+    ssize_t len = readlink("/proc/self/exe", _exePath, NAME_MAX + PATH_MAX +1);
```

Comment:
This should not include the +1 - otherwise assigning the \0 in line 1466 could be off the end.